docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+How to create a package slice for Chisel
+****************************************
+
+If your package doesn't yet have the slice definitions you actually need to **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use :ref:`publishslice` ).
+
+**Let's assume you are trying to create a slice definition for installing the OpenSSL binary into your ROCK!**
+
+Make sure the slice definition doesn't exist yet
+------------------------------------------------
+
+To avoid re-creating a slice that already exists, check the following to see if something that fits your needs already exists:
+
+#. Look into the upstream `chisel-releases <https://github.com/canonical/chisel-releases/tree/ubuntu-22.04/slices>`_ repository
+#. Switch to the branch corresponding to the desired Ubuntu release for your ROCK
+#. Search your package name within the list of slice definitions files
+
+   * if you find it, open it and try to find a slice name containing the bits and pieces you need from that package
+
+
+Structure of a slice definitions file
+------------------------------------
+
+There is **one slice definitions file** for each Ubuntu package. All of the slice definitions files follow the same structure:
+
+.. code-block:: yaml
+
+    # (req) Name of the package.
+    # The slice definition file should be named accordingly (eg. "openssl.yaml")
+
+    package: <package-name>
+
+    # (req) List of slices
+    slices:
+
+        # (req) Name of the slice
+        <slice-name>:
+
+            # (opt) Optional list of slices that this slice depends on
+            essential:
+              - <pkgA_slice-name>
+              - ...
+
+            # (req) The list of files, from the package, that this slice will install
+            contents:
+                </path/to/content>:
+                </path/to/another/multiple*/content/**>:
+                </path/to/moved/content>: {copy: /bin/original}
+                </path/to/link>: {symlink: /bin/mybin}
+                </path/to/new/dir>: {make: true}
+                </path/to/file/with/text>: {text: "Some text"}
+                </path/to/mutable/file/with/default/text>: {text: FIXME, mutable: true}
+                </path/to/temporary/content>: {until: mutate}
+
+            # (opt) Mutation scripts, to allow for the reproduction of maintainer scripts,
+            # based on Starlark (https://github.com/canonical/starlark)
+            mutate: |
+                ...
+
+Find the dependencies of your package
```

Comment:
I like this section, but to me one of the hardest parts of Diátaxis is trying to avoid explaining to much; for how-tos one [should not explain concepts](https://diataxis.fr/how-to-guides/#don-t-explain-concepts) if possible so I actually don't know if this is too much explanation or not! Thoughts?
Maybe we should ask someone on the Documentation team for help? Can either of you attend the Documentation Office Hours today?

Reply:
When are the Documentation office hours ?

Reply:
An hour from this comment; this is the meeting: https://meet.google.com/vpj-sanz-mrb

Reply:
I asked because I can't make it to today's; on second thought it might be better if we finish reviewing this batch and get it merged, consolidate our questions and bring them to Doc office hours next week

Reply:
IMO, this specific section is fine. It doesn't explain stuff, but rather guide you through a process. The one above though, with the slice schema, indeed feels out of place here. I'll comment there